### PR TITLE
Fix/dtrsd-24621

### PR DESCRIPTION
**Pull request by radireddy**
| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #DTRSD-24621
| License          | BSD 3-Clause

## Description

Not possible to change LDP contact. LDP services are not listed on the contact page.

### :bookmark: TODOs

- [x] TRANS-30156

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/account/account.bundle.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import './contacts/service/edit/edit.controller';
 import './contacts/service/edit/edit.routing';
 import './contacts/service/user-contacts-service.controller';
 import './contacts/service/user-contacts-service.routes';
-import './contacts/service/user-contacts.constants';
+import './contacts/user-contacts.constants';
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/account/account.bundle.js:15]: Why moving this file ? You could also target `./service/user-contacts.constants` at https://github.com/ovh/manager/pull/4065/files#diff-0ceb597b8401081cae880cd32fa48a43f1d6a4b9531e5c0c9fef6b8ca5df26b1R4

Comment (radireddy) [reply packages/manager/apps/dedicated/client/app/account/account.bundle.js:15]: I thought it's not good practice for a parent module to depend on the child module

**Comment by ovh-cds** [comment]
CDS Report test#17259.0 ✔ 
* Stage 1
  * lint ✔ 
  * SemVer ✔

**Comment by ovh-cds** [comment]
CDS Report build#17259.0 ✔ 
* 
  * Build ✔

**Comment by ovh-cds** [comment]
CDS Report installation#17259.0 ✔ 
* Stage 1
  * Install ✔

**Review by antleblanc — APPROVED** [review]

**Review by frenautvh — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/account/contacts/user-contacts.constants.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ export const AVAILABLE_SERVICES = [
   'FAILOVER',
   'PACK_SIP_TRUNK',
   'LOAD_BALANCER',
+  'LOGS',
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/account/contacts/user-contacts.constants.js:23]: **thought:** Maybe this list could be sorted next time.

**Review by antleblanc — APPROVED** [review]

**Pull request merged**